test_single_abortable_task failed on INDEXs Pc

First run failed with this message, second was OK, Travis is OK. This commit: https://github.com/jirik/layman/commit/d38d63a1a62261fbe9ac3838bbe3cc5b7b55a6ec
```python
============================= test session starts ==============================
platform linux -- Python 3.6.9, pytest-6.0.1, py-1.9.0, pluggy-0.13.1 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /code
plugins: celery-4.4.7
collected 101 items                                                                                 

src/layman/celery_test.py::test_single_abortable_task FAILED                                                              [  0%]

=========================================================== FAILURES ============================================================
__________________________________________________ test_single_abortable_task ___________________________________________________

client = <FlaskClient <Flask 'layman'>>

    def test_single_abortable_task(client):
        task_names = [
            'layman.layer.filesystem.tasks.refresh_input_chunk',
        ]
        tasks = [
            getattr(
                importlib.import_module(taskname.rsplit('.', 1)[0]),
                taskname.rsplit('.', 1)[1]
            ) for taskname in task_names
        ]
        check_crs = False
        task_options = {
            'crs_id': 'EPSG:4326',
            'description': 'bla',
            'title': 'bla',
            'ensure_user': True,
            'check_crs': check_crs,
        }
        filenames = ['abc.geojson']
        username = 'test_abort_user'
        layername = 'test_abort_layer'
        files_to_upload = input_chunk.save_layer_files_str(username, layername, filenames, check_crs)
        task_chain = chain(*[
            tasks_util._get_task_signature(username, layername, t, task_options, 'layername')
            for t in tasks
        ])
        task_result = task_chain()
    
        results = [task_result]
        results_copy = [
            AbortableAsyncResult(task_result.task_id, backend=celery_app.backend)
            for task_result in results
        ]
    
        time.sleep(1)
    
>       assert results[0].state == results_copy[0].state == 'STARTED'
E       AssertionError: assert 'PENDING' == 'STARTED'
E         - STARTED
E         + PENDING

src/layman/celery_test.py:101: AssertionError
----------------------------------------------------- Captured stdout setup -----------------------------------------------------
 * Serving Flask app "layman" (lazy loading)
 * Environment: development
 * Debug mode: off
==================================================== short test summary info ====================================================
FAILED src/layman/celery_test.py::test_single_abortable_task - AssertionError: assert 'PENDING' == 'STARTED'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================================= 1 failed in 3.30s =======================================================
make: *** [Makefile:141: test] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -144,10 +144,18 @@ celery-worker-test-bash:
 
 test:
 	mkdir -p tmp
-	docker-compose -f docker-compose.deps.yml -f docker-compose.dev.yml build layman_dev
+	if [ "$$(docker images -q layman_dev 2> /dev/null)" = "" ]; then \
+		docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml build layman_dev ; \
+	fi;
+	if [ "$$(docker images -q timgen 2> /dev/null)" = "" ]; then \
+		docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml build timgen ; \
+	fi;
+	if [ "$$(docker images -q layman_client_test 2> /dev/null)" = "" ]; then \
+		docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml build layman_client_test ; \
+	fi;
 	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml rm -f layman_test
-	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml run --rm --no-deps -u root layman_test bash -c "cd src && python3 -B setup_gs_auth.py"
 	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml run --rm --no-deps -u root layman_test bash -c "bash ensure-test-data.sh"
+	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml run --rm --no-deps -u root layman_test bash -c "cd src && python3 -B setup_gs_auth.py"
 	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml up --force-recreate --no-deps -d celery_worker_test
 	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml run --rm --name layman_test_run_1 layman_test
 

--- a/Makefile
+++ b/Makefile
@@ -143,6 +143,7 @@ celery-worker-test-bash:
 	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml run --rm celery_worker_test bash
 
 test:
+	mkdir -p tmp
 	docker-compose -f docker-compose.deps.yml -f docker-compose.dev.yml build layman_dev
 	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml rm -f layman_test
 	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml run --rm --no-deps -u root layman_test bash -c "cd src && python3 -B setup_gs_auth.py"

--- a/Makefile
+++ b/Makefile
@@ -147,6 +147,7 @@ test:
 	docker-compose -f docker-compose.deps.yml -f docker-compose.dev.yml build layman_dev
 	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml rm -f layman_test
 	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml run --rm --no-deps -u root layman_test bash -c "cd src && python3 -B setup_gs_auth.py"
+	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml run --rm --no-deps -u root layman_test bash -c "bash ensure-test-data.sh"
 	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml up --force-recreate --no-deps -d celery_worker_test
 	docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml run --rm --name layman_test_run_1 layman_test
 

--- a/Makefile
+++ b/Makefile
@@ -145,7 +145,7 @@ celery-worker-test-bash:
 test:
 	mkdir -p tmp
 	if [ "$$(docker images -q layman_dev 2> /dev/null)" = "" ]; then \
-		docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml build layman_dev ; \
+		docker-compose -f docker-compose.deps.yml -f docker-compose.dev.yml build layman_dev ; \
 	fi;
 	if [ "$$(docker images -q timgen 2> /dev/null)" = "" ]; then \
 		docker-compose -f docker-compose.deps.yml -f docker-compose.test.yml build timgen ; \

--- a/docker-compose.test.yml
+++ b/docker-compose.test.yml
@@ -19,8 +19,6 @@ services:
         - ./deps/geoserver/sample/geoserver_data:/geoserver/initial_data_dir
       depends_on:
         - celery_worker_test
-        - timgen
-        - layman_client_test
 
    celery_worker_test:
       image: layman_dev:latest
@@ -41,6 +39,8 @@ services:
         - postgresql
         - geoserver
         - micka
+        - timgen
+        - layman_client_test
 
    timgen:
       container_name: timgen

--- a/docker-compose.test.yml
+++ b/docker-compose.test.yml
@@ -15,6 +15,7 @@ services:
         - ./test:/code/test
         - ./tmp:/code/tmp
         - ./test.sh:/code/test.sh
+        - ./ensure-test-data.sh:/code/ensure-test-data.sh
         - ./deps/geoserver/data:/geoserver/data_dir
         - ./deps/geoserver/sample/geoserver_data:/geoserver/initial_data_dir
       depends_on:

--- a/ensure-test-data.sh
+++ b/ensure-test-data.sh
@@ -1,0 +1,63 @@
+#!/bin/bash
+
+set -ex
+
+bash src/clear-python-cache.sh
+
+mkdir -p tmp/naturalearth/110m/cultural
+mkdir -p tmp/naturalearth/50m/cultural
+mkdir -p tmp/naturalearth/10m/cultural
+mkdir -p tmp/data200/trans/jtsk
+mkdir -p tmp/sm5/vektor/jtsk
+
+ne_110m_cultural=tmp/naturalearth/110m_cultural.zip
+if ! [ -f $ne_110m_cultural ]; then
+  curl -L -o $ne_110m_cultural "https://www.naturalearthdata.com/http//www.naturalearthdata.com/download/110m/cultural/110m_cultural.zip"
+  unzip -q $ne_110m_cultural -d tmp/naturalearth/110m/cultural
+fi
+
+ne_110m_cultural_admin_0_countries=tmp/naturalearth/110m/cultural/ne_110m_admin_0_countries.geojson
+if ! [ -f $ne_110m_cultural_admin_0_countries ]; then
+  curl -L -o $ne_110m_cultural_admin_0_countries "https://github.com/nvkelso/natural-earth-vector/raw/master/geojson/ne_110m_admin_0_countries.geojson"
+fi
+
+ne_10m_cultural_admin_0_countries=tmp/naturalearth/10m/cultural/ne_10m_admin_0_countries.geojson
+if ! [ -f $ne_10m_cultural_admin_0_countries ]; then
+  curl -L -o $ne_10m_cultural_admin_0_countries "https://github.com/nvkelso/natural-earth-vector/raw/master/geojson/ne_10m_admin_0_countries.geojson"
+fi
+
+ne_50m_cultural_admin_0_countries=tmp/naturalearth/50m/cultural/ne_50m_admin_0_countries.geojson
+if ! [ -f $ne_50m_cultural_admin_0_countries ]; then
+  curl -L -o $ne_50m_cultural_admin_0_countries "https://github.com/nvkelso/natural-earth-vector/raw/master/geojson/ne_50m_admin_0_countries.geojson"
+fi
+
+ne_110m_cultural_populated_places=tmp/naturalearth/110m/cultural/ne_110m_populated_places.geojson
+if ! [ -f $ne_110m_cultural_populated_places ]; then
+  curl -L -o $ne_110m_cultural_populated_places "https://github.com/nvkelso/natural-earth-vector/raw/master/geojson/ne_110m_populated_places.geojson"
+fi
+
+data200trans=tmp/data200/trans/trans-jtsk.zip
+data200transJtsk=tmp/data200/trans/jtsk
+data200transRoad=tmp/data200/trans/RoadL.shp
+if ! [ -f $data200transRoad ]; then
+  curl -A "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.100 Safari/537.36" -L -o $data200trans "https://geoportal.cuzk.cz/ZAKAZKY/Data200/TRANS.zip"
+  unzip -q $data200trans -d $data200transJtsk
+  for f in tmp/data200/trans/jtsk/RoadL.shp
+  do
+   echo "Processing $f"
+    ogr2ogr -t_srs EPSG:3857 -lco ENCODING=UTF-8 tmp/data200/trans/$(basename $f) $f
+  done
+fi
+
+sm5building=tmp/sm5/vektor/sm5.zip
+sm5VectorJtsk=tmp/sm5/vektor/jtsk
+sm5VectorBuilding=tmp/sm5/vektor/Budova.shp
+if ! [ -f $sm5VectorBuilding ]; then
+  curl -A "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.100 Safari/537.36" -L -o $sm5building "https://geoportal.cuzk.cz/UKAZKOVA_DATA/SM5_VEKTOR_NOVA.zip"
+  unzip -q $sm5building -d $sm5VectorJtsk
+  for f in tmp/sm5/vektor/jtsk/SM5_VEKTOR_NOVA/shp/Budova.shp
+  do
+   echo "Processing $f"
+    ogr2ogr -s_srs EPSG:5514 -t_srs EPSG:3857 -lco ENCODING=UTF-8 tmp/sm5/vektor/$(basename $f) $f
+  done
+fi

--- a/start.sh
+++ b/start.sh
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -ex
+
 bash src/clear-python-cache.sh
 
 

--- a/test.sh
+++ b/test.sh
@@ -1,64 +1,8 @@
 #!/bin/bash
 
+set -ex
+
 bash src/clear-python-cache.sh
-
-mkdir -p tmp/naturalearth/110m/cultural
-mkdir -p tmp/naturalearth/50m/cultural
-mkdir -p tmp/naturalearth/10m/cultural
-mkdir -p tmp/data200/trans/jtsk
-mkdir -p tmp/sm5/vektor/jtsk
-
-ne_110m_cultural=tmp/naturalearth/110m_cultural.zip
-if ! [ -f $ne_110m_cultural ]; then
-  curl -L -o $ne_110m_cultural "https://www.naturalearthdata.com/http//www.naturalearthdata.com/download/110m/cultural/110m_cultural.zip"
-  unzip -q $ne_110m_cultural -d tmp/naturalearth/110m/cultural
-fi
-
-ne_110m_cultural_admin_0_countries=tmp/naturalearth/110m/cultural/ne_110m_admin_0_countries.geojson
-if ! [ -f $ne_110m_cultural_admin_0_countries ]; then
-  curl -L -o $ne_110m_cultural_admin_0_countries "https://github.com/nvkelso/natural-earth-vector/raw/master/geojson/ne_110m_admin_0_countries.geojson"
-fi
-
-ne_10m_cultural_admin_0_countries=tmp/naturalearth/10m/cultural/ne_10m_admin_0_countries.geojson
-if ! [ -f $ne_10m_cultural_admin_0_countries ]; then
-  curl -L -o $ne_10m_cultural_admin_0_countries "https://github.com/nvkelso/natural-earth-vector/raw/master/geojson/ne_10m_admin_0_countries.geojson"
-fi
-
-ne_50m_cultural_admin_0_countries=tmp/naturalearth/50m/cultural/ne_50m_admin_0_countries.geojson
-if ! [ -f $ne_50m_cultural_admin_0_countries ]; then
-  curl -L -o $ne_50m_cultural_admin_0_countries "https://github.com/nvkelso/natural-earth-vector/raw/master/geojson/ne_50m_admin_0_countries.geojson"
-fi
-
-ne_110m_cultural_populated_places=tmp/naturalearth/110m/cultural/ne_110m_populated_places.geojson
-if ! [ -f $ne_110m_cultural_populated_places ]; then
-  curl -L -o $ne_110m_cultural_populated_places "https://github.com/nvkelso/natural-earth-vector/raw/master/geojson/ne_110m_populated_places.geojson"
-fi
-
-data200trans=tmp/data200/trans/trans-jtsk.zip
-data200transJtsk=tmp/data200/trans/jtsk
-data200transRoad=tmp/data200/trans/RoadL.shp
-if ! [ -f $data200transRoad ]; then
-  curl -A "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.100 Safari/537.36" -L -o $data200trans "https://geoportal.cuzk.cz/ZAKAZKY/Data200/TRANS.zip"
-  unzip -q $data200trans -d $data200transJtsk
-  for f in tmp/data200/trans/jtsk/RoadL.shp
-  do
-   echo "Processing $f"
-    ogr2ogr -t_srs EPSG:3857 -lco ENCODING=UTF-8 tmp/data200/trans/$(basename $f) $f
-  done
-fi
-
-sm5building=tmp/sm5/vektor/sm5.zip
-sm5VectorJtsk=tmp/sm5/vektor/jtsk
-sm5VectorBuilding=tmp/sm5/vektor/Budova.shp
-if ! [ -f $sm5VectorBuilding ]; then
-  curl -A "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.100 Safari/537.36" -L -o $sm5building "https://geoportal.cuzk.cz/UKAZKOVA_DATA/SM5_VEKTOR_NOVA.zip"
-  unzip -q $sm5building -d $sm5VectorJtsk
-  for f in tmp/sm5/vektor/jtsk/SM5_VEKTOR_NOVA/shp/Budova.shp
-  do
-   echo "Processing $f"
-    ogr2ogr -s_srs EPSG:5514 -t_srs EPSG:3857 -lco ENCODING=UTF-8 tmp/sm5/vektor/$(basename $f) $f
-  done
-fi
 
 mkdir -p tmp/artifacts
 rm -rf tmp/artifacts/*

